src/Components/Navbar.js

```diff
@@ -1,42 +1,53 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import "./Navbar.css";
-import { FaSearch } from "react-icons/fa";
 import Logo from "./assets/Logo.png";
 import Carousel from "./Carousel";
 import LoginButton from "./login";
-import { useAuth0 } from "@auth0/auth0-react";
+import axios from "axios";
 
 export default function Navbar() {
-  const { isAuthenticated, user } = useAuth0();
+  const [search, setSearch] = useState("");
+  const [FundsData, setFundsData] = useState(null);
 
+  useEffect(() => {
+    axios.get("http://localhost:8000/apis/v1/funds").then((data) => {
```

Comment:
Add `http://localhost:8000` in `.env` here, you should keep it this way so you don't have to change it again when you deploy it.